modules/@apostrophecms/schema/lib/addFieldTypes.js

```diff
@@ -96,7 +96,7 @@ module.exports = (self) => {
   self.addFieldType({
     name: 'string',
     convert(req, field, data, destination) {
-      destination[field.name] = self.apos.launder.string(data[field.name], field.def);
+      destination[field.name] = self.apos.launder.string(data[field.name], destination[field.name] || undefined);
```

Comment:
I see we need some nuance here. For the basic form input fields, we don't want to go all the way back to undefined, because the empty string is a legitimate value for a string.

So we don't want undefined here, in fact we don't need an `||` clause at all. But we still want to get rid of passing `field.def` here because it is not our job (it is the job of `newInstance` and similar code).